.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 *.log
 
-deploys
+.deploys
```

Comment:
Added "." to deploy directory so the directory is always at top of Atom tree-view.